PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -297,7 +297,11 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     src = cms.InputTag("linkedObjects","electrons"),
     name= cms.string("Electron"),
     doc = cms.string("slimmedElectrons after basic selection (" + finalElectrons.cut.value()+")"),
-    variables = cms.PSet(CandVars,
+    variables = cms.PSet(), ## PSet defined below in mode-dependent way
```

Comment:
please leave the content of `variables` the default one, and extend it with the `_eleVarsExtra` and `` ; i.e. modifying `electronTable` on the spot in the customise function